README.md

```diff
@@ -90,3 +90,22 @@ Please note that by default there are always ignored errors with these keywords:
 Thiey are ignored because sometimes project's nginx config file may contain `include` directives to files without their real presence in the repo (these files are added as [nginx defaults](https://github.com/nginx/nginx/tree/master/conf) during installation) and can be ignored during pre-commit hook processing.
 
 This is it!
+
+### `add_task_number`
```

Comment:
Could you please add example of what should be added to `.pre-commit-config` to make this hook work? Including default parameters for regex and format